app.R

```diff
@@ -358,7 +362,19 @@ server <- function(input, output, session) {
   # Render the race results table
   output$race_results_table <- DT::renderDT({
     
-    datatable(filtered_race_results())
+    datatable(filtered_race_results(),
+              options = list("pageLength" = 20,
+                             "paging" = FALSE,
+                             "scrollY" = '800px',
+                             "autoWidth" = TRUE
```

Comment:
Add `columnDefs = list(list(visible = FALSE, targets = c("flag")))` within the list for `options` will make the flag column not show up.

Reply:
Cool, this works